src/enums/filingTypes.ts

```diff
@@ -18,6 +18,8 @@ export enum FilingTypes {
   SPECIAL_RESOLUTION = 'specialResolution',
   TRANSITION = 'transition',
   VOLUNTARY_DISSOLUTION = 'voluntaryDissolution',
+  ADMINISTRATIVE_DISSOLUTION = 'administrativeDissolution',
+  PUT_BACK_ON = 'putBackOn',
```

Comment:
Are there any mixins methods to update? `isTypeXXX' and `filingTypeToName`

Reply:
added mixins methods for new Filing Types